api/src/main/java/org/geysermc/geyser/api/connection/GeyserConnection.java

```diff
@@ -75,6 +75,13 @@ public interface GeyserConnection extends Connection, CommandSource {
      */
     int protocolVersion();
 
+    /**
+     * Sends a command as if the player had executed it.
+     *
+     * @param command the command without the leading forward-slash
+     */
+    void executeCommand(String command);
```

Comment:
let's rename this to sendCommand - execute might imply that it forcibly runs the command; but this isnt a given if e.g. perms arent present

the internal / existing method could be renamed to sendCommandPacket or similar